frontend/src/metabase-lib/parameters/utils/parameter-values.js

```diff
@@ -28,6 +28,15 @@ export function getDefaultValuePopulatedParameters(
   });
 }
 
+// Needed because parameter values might be arrays
+// in which case order of elements isn't guaranteed
+export function isParameterValuesIdentical(a, b) {
```

Comment:
Nit: wouldn't it be more correct to call this `areParameterValuesIdentical`?

Reply:
Yeah, I agree. I am just too used to calling booleans with `is`.